tests/py/dynamo/backend/test_partitioning.py

```diff
@@ -30,6 +30,52 @@ def forward(self, x, y):
             "Single operators should not be segmented",
         )
 
+    def test_partition_fully_supported_one_op_require_full_compilation(self):
```

Comment:
How about we move test_partitioning out of backend directory and test with both export and compile ?

Reply:
Ok that sounds good. These test cases already test just the partitioners and not the `torch.compile` path explicitly, so the tests will work for both paths as-is. Should I make a new test directory `shared_utilities`, containing `test_decompositions`, `test_partitioning`, and `test_compiler_utils`?

Reply:
sounds good